app/helpers/admin/orders_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Admin::OrdersHelper
+  def get_status
```

Comment:
```suggestion
  def load_statuses
```

đổi tên đi em, chị nhớ từng bảo ko nên bắt đầu tên methdo là set.get mà